Library/Homebrew/version.rb

```diff
@@ -179,6 +179,19 @@ def self.detect(url, specs)
     end
   end
 
+  # TODO add tests if method is going to be left
+  def self.parse_version(val)
```

Comment:
Personally I would prefer a universal interface, so anyone who wasn't familiar with internal implementation can easy use this interface and won't be confusing on two seemly similar methods.